test/JDBC/input/views/sys-all_parameters-vu-prepare.sql

```diff
@@ -0,0 +1,153 @@
+CREATE PROCEDURE sys_all_parameters_vu_prepare_addTwo @num1 int, @num2 int
```

Comment:
all the views which are recreated in upgrade script should have dependency tests based on reference in view/function/procedure